pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -159,7 +158,7 @@ func (exp *flowExporter) initFlowExporter(collector net.Addr) error {
 	exp.process = expProcess
 	exp.templateID = expProcess.NewTemplateID()
 
-	expProcess.LoadRegistries()
+	ipfixregistry.LoadRegistry()
```

Comment:
Can you create an interface for this in flowexporter/ipfix pkg so that antrea unit tests can be independent of go-ipfix library by using mock for that interface? 
In that interface you could add load registry, getInfoElement methods?